docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/homelanding/Homelanding.scss

```diff
@@ -523,10 +523,10 @@ parent accordion in all breakpoints. */
 }
 .multimedia-feature-card,
 .multimedia .feature-card {
-  @extend %autoheight-fix;
-  
+  //@extend %autoheight-fix;
   a {
```

Comment:
I was under the assumption based on @BrendanBeltzNIH comment that adding the class back in would not need a change to Homelanding.scss. Was that incorrect?

Reply:
True, it does no harm but just ends up being bloat .  This is just rolling it back to what it was already.  I believe Brendan just wanted to add the wrapper div back to the template and the style is already handled with what's there now.

Reply:
Cool!